metrics/config.go

```diff
@@ -144,13 +144,19 @@ func NewStackdriverClientConfigFromMap(config map[string]string) *StackdriverCli
 	}
 }
 
-// Record applies the `ros` Options to `ms` and then records the resulting
+// RecordBatch applies the `ros` Options to each measurement in `mss` and then records the resulting
 // measurements in the metricsConfig's designated backend.
-func (mc *metricsConfig) Record(ctx context.Context, ms stats.Measurement, ros ...stats.Options) error {
+func (mc *metricsConfig) RecordBatch(ctx context.Context, mss []stats.Measurement, ros ...stats.Options) error {
```

Comment:
What do you think about making this method non-exported? I don't think it's called anywhere outside this package.

Reply:
Actually, even better, I don't think `*metricsConfig.Record` is called at all outside the package, so we should only need one of these. (This exists simply to support the `nil` mc and default `recorder` case.)